src/Explorer/Controls/SmartUi/SmartUiComponent.tsx

```diff
@@ -331,7 +331,6 @@ export class SmartUiComponent extends React.Component<SmartUiComponentProps, Sma
           onChange={(_, item: IDropdownOption) => this.props.onInputChange(input, item.key.toString())}
           placeholder={this.props.getTranslation(placeholderTKey)}
           disabled={disabled}
-          dropdownWidth="auto"
```

Comment:
@srinathnarayanan FYI, not sure if it affects your stuff, since you're UI is generic.

Reply:
dropdownWidth props accept number type. So In case of auto, Its give eslint and typescript error. Default value accept 0.

![image](https://user-images.githubusercontent.com/79906609/112454781-0ccc2300-8d7f-11eb-97e9-ed3dd373ecaf.png)

Reply:
IDropdownProps.dropdownWidth?: number | "auto"
this is available from office-ui-fabric-react v "7.164.2"
So i am not sure what you are showing here

Reply:
@srinathnarayanan Actually When I was trying to push with dropdownWidth="auto", It was giving typescript error. That is why I just check type and removed.

Reply:
did you do an npm install after syncing with master? Check your version of office-ui-fabric-react

Reply:
I think, The issue was with local cache of node_modules. I have updated with master and install npm install. But it was not working. But when I delete node_modules and again install it was working. I revert my changes of this file.
Thank you for suggestion.